cypress/e2e/add-event.cy.ts

```diff
@@ -0,0 +1,157 @@
+/* eslint-disable @typescript-eslint/unbound-method */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+import ProjectLinks from "helpers/project-links";
+
+describe("Add dataset's metadata", () => {
+    beforeEach(() => {
+        cy.visit(ProjectLinks.URL_HOME);
+        const date = new Date().getTime();
+        cy.get('[data-test-id="addNewBlock"]').click();
+        cy.get('[data-test-id="dataset-name"]').type(
+            `mock-root-dataset-${date}`,
+        );
+        cy.get('[data-test-id="create-dataset"]').click();
+    });
+
+    xit("Should check add license of the  dataset", () => {
```

Comment:
Please put a comment why test is excluded

Reply:
Left comment